.travis/linux.debian-sid.script.sh

```diff
@@ -31,6 +31,40 @@ else
 	sudo pbuilder --update --basetgz "$BASETGZ"
 fi
 
+sync_version() {
+	local VERSION
+	local MMR
+	local STAGE
+	local EXTRA
```

Comment:
`shellcheck` says `local` is undefined in the POSIX shell. Most shells(bash, dash, etc.) support the keyword though.
Since `master` checks shell scripts during CI builds using `shellcheck`, we have three choices(`stable-1.2` isn't affected):
- Specify a shell in the shebang which supports `local`
- Do nothing but silence `shellcheck` warnings
- Use a portable way for declaring local variables, if any